editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -1472,7 +1472,7 @@ void Node3DEditorViewport::_sinput(const Ref<InputEvent> &p_event) {
 
 				Vector3 ray_pos = _get_ray_pos(m->get_position());
 				Vector3 ray = _get_ray(m->get_position());
-				float snap = EDITOR_GET("interface/inspector/default_float_step");
+				float snap = EDITOR_GET("interface/inspector/default_step_size");
```

Comment:
This line should be `double snap`.

Reply:
Updated `snap` to double. Also had to update `angle` further down to fix a compile warning. 

Regarding the name change, in hindsight I'm not to keen on the new name. I think `default_precision` or `default_resolution` would be more descriptive for how the variable is meant to be used. At the very least, I think having float in the name is misleading, but I'll implement whatever you guys think is best.

Reply:
In Godot's API, `float` is a 64-bit float point type, so the name was fine IMO.

But if we want to change it, it could be `default_decimal_resolution` yes.